### PR TITLE
feat: [OENG-12938]: Integrating Error Tracking into Platform 

### PR DESCRIPTION
**Pull request by ccaspanello**
##### Summary:

Integrating OverOps (now Error Tracking) into plaform

##### Jira Links:

https://harness.atlassian.net/browse/OENG-12938

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by ccaspanello** [comment]
retrigger titlecheck

**Comment by ccaspanello** [comment]
retrigger prettier

**Comment by ccaspanello** [comment]
retrigger titlecheck

**Comment by ccaspanello** [comment]
retest

**Comment by ccaspanello** [comment]
retrigger jest

File: configs/modulefederation.config.js [hunk 1]
```diff
@@ -33,14 +32,18 @@ module.exports = ({ enableGovernance, enableGitOpsUI, enableSTO }) => {
...
-  // if (enableGovernance) {
   remotes.governance = "governance@[window.getApiBaseUrl('pm/remoteEntry.js')]"
-  // }
 
   if (enableSTO) {
     remotes.sto = "sto@[window.getApiBaseUrl('sto/remoteEntry.js')]"
   }
-
+  
+  if(process.env.TARGET_LOCALHOST){
```
Comment (tan-nhu) [inline configs/modulefederation.config.js:41]: apply formatter

**Review by tan-nhu — APPROVED** [review]

**Review by DFieldFL — DISMISSED** [review]

File: configs/modulefederation.config.js (hunk 1 above)
Comment (ccaspanello) [reply configs/modulefederation.config.js]: Done

**Review by anjanBalgovind — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Comment by ccaspanello** [comment]
retrigger jest

**Comment by ccaspanello** [comment]
retrigger jest

**Comment by ccaspanello** [comment]
retrigger coverage

**Pull request merged**